routes/highSchoolGraduates.js

```diff
@@ -1,8 +1,8 @@
 const express = require('express');
-const createHighSchoolGraduates = require('../controllers/highSchoolGraduates/createHighSchoolGraduatesController');
-const getHighSchoolGraduates = require('../controllers/highSchoolGraduates/getHighSchoolGraduatesController');
-const getManyHighSchoolGraduates = require('../controllers/highSchoolGraduates/getManyHighSchoolGraduatesController');
-const deleteHighSchoolGraduates = require('../controllers/highSchoolGraduates/deleteHighSchoolGraduatesController')
```

Comment:
The folder in controllers is currently named "highSchoolGraduates" so these should remain the same